Modules/Forum/classes/class.ilForumCronNotification.php

```diff
@@ -73,6 +92,35 @@ public function __construct(\ilDBInterface $database = null, \ilForumNotificatio
 			$notificationCache = new \ilForumNotificationCache();
 		}
 		$this->notificationCache = $notificationCache;
+
+		if ($ilAccess === null) {
+			global $DIC;
+			$ilAccess = $DIC->access();
+		}
+		$this->ilAccess = $ilAccess;
+
+		if ($language === null) {
+			global $DIC;
```

Comment:
@legionth If using `$DIC` in the constructor really works (see my comment above), if prefer one usage of `global $DIC;`.